### DISCUSSION TITLE
Upgrade to Vue 3

### DISCUSSION THREAD

**Discussion by oplexz** [wg-easy/wg-easy#747] [Ideas]
I would like to suggest upgrading the project from Vue 2 to Vue 3.

Vue 3 offers several advantages over Vue 2, including improved performance, TypeScript support, and new features like the Composition API, Suspense, Teleport, and so on. Additionally, Vue 3 is smaller in size, which can lead to faster load times. 

Is there a plan to upgrade to Vue 3 in the near future?

**Comment by peterlewis** [comment]
There certainly ain’t a plan to *not* update. Please feel free to submit PR and we’d be happy to work with you on it. Thanks!

**Comment by pheiduck** [comment]
There is an upgrade path: https://www.npmjs.com/package/@vue/compat

**Comment by pheiduck** [comment]
@oplexz, is it okay for you if we bring your commit upstream?
https://github.com/oplexz/wg-easy/commit/1754bae659f82e1f2fb8f406195fcab572355434
